src/services/totalsService.js

```diff
@@ -22,12 +22,11 @@ class TotalsService extends EventTargetShim {
 
     startDate = startDate ? format(startDate, 'yyyy-MM-dd') : format(addDays(new Date(), -14), 'yyyy-MM-dd');
     endDate = endDate ? format(endDate, 'yyyy-MM-dd') : format(addDays(new Date(), 56), 'yyyy-MM-dd');
-
     const cacheKey = `${startDate}${endDate}`;
-
+    this.lastCacheKey = cacheKey;
     if (opts.forceRefresh || this.cache[cacheKey]?.status === 'failed' || !this.cache[cacheKey]) {
       try {
-        this.cache[cacheKey] = {};
+        // this.cache[cacheKey] = {};
```

Comment:
I think that we need to reset the object in cache before making request

Reply:
It's a singleton, if we delete the cache we drop the current logic, I think

Reply:
this.cache[cacheKey] contains result from API. But yeah, I see that it's not so important. In any way cache value will have result (result from API or in exception handler).